astronomer_starship/providers/starship/hooks/starship.py

```diff
@@ -71,11 +73,21 @@ def set_task_instances(self, task_instances: list):
 class StarshipLocalHook(BaseHook, StarshipHook):
     """Hook to retrieve local Airflow data, which can then be sent to the Target Starship instance."""
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._starship_compat = None
```

Comment:
nit: 
(and I'm uncertain)
could you skip the init here? It's just marking that there's a property `_starship_compat`, right? and initializing to `None` ?